elfpy/markets/hyperdrive/hyperdrive_market.py

```diff
@@ -409,6 +417,7 @@ def close_short(
         mint_time: float,
     ) -> tuple[hyperdrive_actions.MarketDeltas, wallet.Wallet]:
         """Calculate the deltas from closing a short and then update the agent wallet & market state"""
+        # calc market and agent deltas
```

Comment:
there should be an `apply_checkpoint()` in here too.  open/close short/long and add/remove liquidity should all have it.

Reply:
I fail tests when I include it. It ends up adjusting the volumes twice.

Reply:
Notice that in the original version, `apply_checkpoint` did not exist in this func